### DISCUSSION TITLE
What's purpose of ~/.podman-desktop dir on macos?

### DISCUSSION THREAD

**Discussion by tony-sol** [podman-desktop/podman-desktop#1699] [Q&A] [answered]
After each startup of podman-desktop, there is an empty `.podman-desktop` appear

<img width="528" alt="tree ~/.podman-desktop" src="https://user-images.githubusercontent.com/33671815/225109509-3c2214f9-c8f9-4447-b4bf-b601a8bb4704.png">

In source code, i can't find why and when it've been created

**Comment by benoitf** [comment, marked as answer]
It was due to an invalid subfolder usage
https://github.com/containers/podman-desktop/issues/1429

**Reply by tony-sol** [reply]
LOL, i read that issue, but totally forgot one 🤣
